roxctl/central/backup/backup.go

```diff
@@ -46,13 +46,16 @@ You can use it to restore central service and the database.`,
 If the provided path is a file path, the backup will be written to the file, overwriting it if it already exists. (The directory MUST exist.)
 If the provided path is a directory, the backup will be saved in that directory with the server-provided filename.
 If this argument is omitted, the backup will be saved in the current working directory with the server-provided filename.`)
+	c.Flags().BoolVar(&centralBackupCmd.certsOnly, "certsOnly", false, `only backs up the certs.
```

Comment:
We strive to use kebap case for CLI flags, so let's make this `certs-only`.

(This is also a TODO for me to add this section in [the roxctl README](https://github.com/stackrox/stackrox/blob/master/roxctl/README.md)).